aws/cluster/networking.tf

```diff
@@ -2,22 +2,9 @@
  * Configures a VPC on top of which we'll use kops to spawn our Kubernetes cluster
  */
 
-// The VPC itself
-resource "aws_vpc" "main" {
-  cidr_block           = "10.${var.vpc-number}.0.0/16"
```

Comment:
Should we make this switchable (enable/disable) in case people want to use this module to make their VPCs? Perhaps we can do it later.

Reply:
Yes I wanted to do that but actually didn't found any way how to do it because of that non-lazy evaluation of variables. I used count to (not) provision the VPC or use ID provided. But then errors were raised during planning phase as it told me `aws_vpc.main.id` does not exist even in a case where it was not needed. Like here:
```
vpc_id = "${var.create-vpc == "true" ? aws_vpc.main.id : var.vpc-id}
```
Even when `var.create-vpc` was false terraform complained that it does not know `aws_vpc.main.id`. 

It is Terraform open issue https://github.com/hashicorp/terraform/issues/11574